modules/admin-ui/public/env-config.js

```
@@ -0,0 +1 @@
+window.__env__ = {};
```

default, to avoid misleading error messages